capitalize_decor.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decor'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] the `upcase` make all the characters in upper case, but in the requirements, we only required to capitalize the first letter, so kindly use the `capitalize` instead:

`@nameable.correct_name.capitalize`